experimental/dds/tree2/src/test/shared-tree/fuzz/topLevelFuzz.spec.ts

```diff
@@ -3,51 +3,27 @@
  * Licensed under the MIT License.
  */
 import path from "path";
+import { takeAsync } from "@fluid-internal/stochastic-test-utils";
+import { DDSFuzzModel } from "@fluid-internal/test-dds-utils";
 import {
-	AsyncGenerator,
-	makeRandom,
-	performFuzzActionsAsync,
-	SaveInfo,
-} from "@fluid-internal/stochastic-test-utils";
-import { TestTreeProvider, SummarizeType, initializeTestTree } from "../../utils";
-import {
-	FuzzTestState,
-	makeOpGenerator,
-	makeOpGeneratorFromFilePath,
-	EditGeneratorOpWeights,
-} from "./fuzzEditGenerators";
-import { checkTreesAreSynchronized, fuzzReducer } from "./fuzzEditReducers";
-import { initialTreeState, runFuzzBatch, testSchema } from "./fuzzUtils";
+	createDDSFuzzSuite,
+	DDSFuzzTestState,
+	// eslint-disable-next-line import/no-internal-modules
+} from "@fluid-internal/test-dds-utils/dist/ddsFuzzHarness";
```

Comment:
same thing here--import can be from the main index.

Reply:
updated imports